CSVSource/build.gradle

```diff
@@ -78,3 +78,31 @@ tasks.withType(Test) {
         systemProperty 'EntFullFilePath', rootProject.findProperty('EntFullFilePath')?:"b.csv"
     }
 }
+
+// Using gradle-dcompose-plugin to build CSVSource docker image
+dcompose {
+    if (rootProject.hasProperty('dockerRegistry')) {
+        def registryName = rootProject.getProperty('dockerRegistry')
+        csvSource {
+            buildFiles = project.copySpec {
+                from "$projectDir/src/main/docker/"
+                from "$projectDir/build/distributions/CSVSource.tar"
+            }
+            if (rootProject.hasProperty('csvImageTag')) {
+                repository = registryName + '/csv-source' + rootProject.getProperty('csvImageTag')
+            } else {
+                repository = registryName + '/csv-source'
+            }
+            logBuildStatus = true
+            buildLogFile = file("$buildDir/csvDockerBuildLog.file")
+        }
```

Comment:
Similar -- though here since they all go into their own build areas, can't it just be `dockerBuildLog.txt` (`.txt` rather than `.file` unless there's a good reason for that.   Then double clicking on it does something reasonable.